dist/darkly/_variables.scss

```diff
@@ -147,6 +147,11 @@ $modal-content-border-color:        $gray-700 !default;
 
 $modal-header-border-color:         $gray-700 !default;
 
+// Toasts
+$toast-background-color:            $gray-700 !default;
+
```

Comment:
I've also added `$toast-border-color: $gray-700 !default;`.

The Card component uses the `$gray-700` color for its border while Bootstraps' default border color for Toasts is `rgba(0, 0, 0, .1)` (see https://github.com/twbs/bootstrap/blob/v4.4.1/scss/_variables.scss#L914).